config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -285,6 +285,46 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: node-kubelet-flaky
+- name: ci-kubernetes-node-kubelet-flaky-kubetest2
+  interval: 2h
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  decoration_config:
+    timeout: 60m
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: node-kubelet-flaky-kubetest2
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211124-2ed05120f3-experimental
+      env:
+      - name: GOPATH
+        value: /go
+      command:
+      - runner.sh
+      args:
+      - kubetest2
+      - noop
+      - --test=node
+      - --
+      - --repo-root=.
+      - --gcp-project-type=node-e2e-project
```

Comment:
Can we avoid adding new things to this project?

Reply:
I've added to this project as the non-kubetest2 job was using it. Do we have any other options?

Reply:
FWIW just to clarify this is not a single project, but a pool of projects
https://github.com/kubernetes/test-infra/blob/5dd9ad002f35335e89b1cbc613b7f5695d181af9/config/prow/cluster/build/boskos-resources/boskos-resources.yaml#L329-L349 
and one project from it is leased through [boskos](https://github.com/kubernetes-sigs/boskos), at runtime during each job.

The name of the pool is intentionally designed to easily identify the types of jobs using that pool. So I would say `node-e2e-project` is the right choice for e2e node tests. 
If we see load being an issue due to the new jobs being added, we might want to consider adding more projects to the pool.